src/Microsoft.IdentityModel.Tokens/CallContext.cs

```diff
@@ -48,9 +48,6 @@ public CallContext()
         /// </summary>
         public CallContext(Guid activityId)
         {
-            if (activityId == null)
-                throw new ArgumentNullException(nameof(activityId));
```

Comment:
should we instead set it to empty or skip setting the value? Why potentially setting it to null? Also not sure why it's not ok to throw here, if the developer doesn't want to set the activity id they should use the base ctr?

Reply:
Guid is a value type, so it could never be null, This came up from sonar, marked like 52 as non-reachable code